client/gutenberg/extensions/vr/editor.js

```diff
@@ -1,34 +1,9 @@
 /** @format */
-/**
- * External dependencies
- */
-import { registerBlockType } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
-import './editor.scss';
-import VRImageEdit from './edit';
-import VRImageSave from './save';
-import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import registerJetpackBlock from 'gutenberg/extensions/presets/jetpack/utils/register-jetpack-block';
+import { name, settings } from '.';
 
-registerBlockType( 'jetpack/vr', {
-	title: __( 'VR Image' ),
-	description: __( 'Embed 360° photos and Virtual Reality (VR) content' ),
-	icon: 'embed-photo',
-	category: 'jetpack',
-	keywords: [ __( 'vr' ), __( 'panorama' ), __( '360' ) ],
-	supports: {
-		html: false,
-	},
-	attributes: {
-		url: {
-			type: 'string',
-		},
-		view: {
-			type: 'string',
-		},
-	},
-	edit: VRImageEdit,
-	save: VRImageSave,
-} );
+registerJetpackBlock( name, settings );
```

Comment:
Seems like this should use `registerBlockType()` here - there is no VR module, so the VR block should always be registered.

Reply:
Hmm, thinking about it some more -- we should address this on the JP side, since otherwise, the VR block will always be visible (regardless of plan or other criteria). We have precedent -- there's no Map module either, so we're registering it in https://github.com/Automattic/jetpack/blob/b25ebc43d48f32d587f9a5c8b9ab4c4f96e3767a/modules/blocks.php

Reply:
That's fair, let's do that on the Jetpack side.

Reply:
Jetpack PR to register it on the server side: https://github.com/Automattic/jetpack/pull/10934